tests/microos/toolbox.pm

```diff
@@ -37,7 +37,7 @@ sub create_user {
     assert_script_run "grep '^${serial_group}:.*:${user}\$' /etc/group || (chown $user /dev/$testapi::serialdev && gpasswd -a $user $serial_group)";
 
     # Don't ask password for sudo commands
-    assert_script_run "echo \"$user ALL=(ALL) NOPASSWD:ALL\" >> /etc/sudoers";
+    assert_script_run "echo \"$user ALL=(ALL) NOPASSWD:ALL\n\" >> /etc/sudoers.d/50-$user.conf";
```

Comment:
```suggestion
    assert_script_run "echo -e '$user ALL=(ALL) NOPASSWD:ALL\n' >> /etc/sudoers.d/50-$user.conf";
```

Why double newline?

Reply:
I thought that the missing newline was the reason why the VR failed - turns out it's not - even with this, it still fails (so I will certainly revert that part - but need to figure out what is wrong with the sudo config)